block.go

```diff
@@ -162,6 +162,14 @@ type BlockStats struct {
 	NumSeries     uint64 `json:"numSeries,omitempty"`
 	NumChunks     uint64 `json:"numChunks,omitempty"`
 	NumTombstones uint64 `json:"numTombstones,omitempty"`
+	NumBytes      int64  `json:"numBytes,omitempty"`
+}
+
+// BlockDesc describes a block by ULID and time range.
```

Comment:
You need to remove L168-174 to fix the test.

Reply:
Not quite sure how I missed that, thanks though

Reply:
This test will still fail though because of this test in the `TestDB_reloadOrder` function: 
```
	testutil.Equals(t, *metas[1], blocks[0].Meta())
``` 
on line 92, because after it reloads the block it calculates the size. Should I go in and modify that test then so it checks against the correct size?